numpy/random/mtrand/distributions.c

```diff
@@ -315,7 +316,7 @@ long rk_binomial_btpe(rk_state *state, long n, double p)
     v = v*(u-p3)*lamr;
 
   Step50:
-    k = fabs(y - m);
+    k = labs(y - m);
```

Comment:
Hmm, k is `long`, also gets squared further down (overflow?). I quess this is no worse than it was, but the whole routine might be less than perfect.